### PR TITLE
Upgrade to latest upstream 1.8.1

### PR DESCRIPTION
**Pull request by Venemo**

### CODE REVIEW

File: rpm/gtest.spec [hunk 1]
```diff
@@ -1,126 +1,137 @@
...
-Release:        0
-License:        GPL-2.0+
-Summary:        Google C++ Testing Framework
-Url:            https://code.google.com/p/googletest
-Group:          Definition/Libraries/C and C++
-Source:         %{name}-%{version}.tar.bz2
-Patch0:         gtest-soname.patch
-
+Version:        1.8.1
+Release:        3%{?dist}
```
Comment (pvuorela) [inline rpm/gtest.spec]: This looks like Fedora. Can be just 1.

**Comment by pvuorela** [comment]
Bug reference?

**Comment by uaoh** [comment]
It shouldn't hurt to squash the commits.

**Comment by xfade** [comment]
> 
> 
> It shouldn't hurt to squash the commits.

'Squash and merge' option takes care of that

File: rpm/gtest.spec (hunk 1 above)
Comment (Venemo) [reply rpm/gtest.spec]: Fixed.

**Pull request merged**